website/views/shared/header/header.jade

```diff
@@ -31,9 +31,10 @@
           | {{Math.floor(user.stats.mp)}} / {{user._statsComputed.maxMP}}
     // party
     .party(ng-controller='PartyCtrl')
-      button.party-invite.btn.btn-primary(ng-click="inviteOrStartParty(group)",
-        ng-if="(!party.members || party.memberCount === 1) && user.preferences.displayInviteToPartyWhenPartyIs1",
-        popover="{{!party.members ? env.t('startAParty') : env.t('addToParty')}}", popover-placement="left", popover-trigger="mouseenter")
-        span=env.t("battleWithFriends")
       .herobox-wrap(ng-repeat='profile in partyMinusSelf')
         +herobox()
+      .party-invite-wrap
+        button.party-invite.btn.btn-primary(ng-click="inviteOrStartParty(group)",
+          ng-if="(!party.members || party.memberCount < 4 ) && user.preferences.displayInviteToPartyWhenPartyIs1",
```

Comment:
I'm in favour of changing it and I'm willing to do the migration.  The more accurate we make our names, the less pain we'll suffer later.

Perhaps `displayInviteToPartyWhenPartyIsSmall` to avoid future problems of this nature.